index.js

```diff
@@ -270,6 +274,18 @@ const generateReport = async options => {
     data.previousPeriod = json.settings.length > 0 ? json.settings[0].value : '';
   }
 
+  if (options.coverage) {
+    const response = await got(
+      `${sonarBaseURL}/api/measures/component?component=${sonarComponent}&metricKeys=coverage${filterCoverage}`,
```

Comment:
question: does this request works for every sonarqube version ?

Reply:
Since 5.4. Additionnally, query param `pullRequest` is available since 7.1.